transaction_record.go

```diff
@@ -65,6 +68,163 @@ type TransactionRecord struct {
+	}
+
+	customFeesJSON := make([]AssessedCustomFeeJSON, len(record.AssessedCustomFees))
+
+	for i, fee := range record.AssessedCustomFees {
+		payerAccountIDsStr := make([]string, len(fee.PayerAccountIDs))
+		for j, accID := range fee.PayerAccountIDs {
+			payerAccountIDsStr[j] = accID.String()
+		}
+		customFeesJSON[i] = AssessedCustomFeeJSON{
+			FeeCollectorAccountID: fee.FeeCollectorAccountId.String(),
+			TokenID:               fee.TokenID.String(),
+			Amount:                fmt.Sprint(fee.Amount),
+			PayerAccountIDs:       payerAccountIDsStr,
+		}
+	}
+
+	m["assessedCustomFees"] = customFeesJSON
+
+	type TokenAssociationOutput struct {
+		TokenID   string `json:"tokenId"`
+		AccountID string `json:"accountId"`
+	}
+	automaticTokenAssociations := make([]TokenAssociationOutput, len(record.AutomaticTokenAssociations))
+	for i, ta := range record.AutomaticTokenAssociations {
+		automaticTokenAssociations[i] = TokenAssociationOutput{
+			TokenID:   ta.TokenID.String(),
+			AccountID: ta.AccountID.String(),
+		}
+	}
+	m["automaticTokenAssociations"] = automaticTokenAssociations
+
+	consensusTime := record.ConsensusTimestamp.UTC().Format("2006-01-02T15:04:05.000Z")
+	parentConsesnusTime := record.ParentConsensusTimestamp.UTC().Format("2006-01-02T15:04:05.000Z")
+	m["consensusTimestamp"] = consensusTime
+	m["parentConsensusTimestamp"] = parentConsesnusTime
+
+	m["aliasKey"] = fmt.Sprint(record.AliasKey)
+	m["ethereumHash"] = hex.EncodeToString(record.EthereumHash)
+	type PaidStakingReward struct {
+		AccountID  string `json:"accountId"`
+		Amount     string `json:"amount"`
+		IsApproved bool   `json:"isApproved"`
+	}
+	var paidStakingRewards []PaidStakingReward
+	for k, reward := range record.PaidStakingRewards {
+		paidStakingReward := PaidStakingReward{
+			AccountID:  k.String(),
+			Amount:     fmt.Sprint(reward.AsTinybar()),
+			IsApproved: false,
+		}
+		paidStakingRewards = append(paidStakingRewards, paidStakingReward)
+	}
+
+	sort.Slice(paidStakingRewards, func(i, j int) bool {
+		return paidStakingRewards[i].AccountID < paidStakingRewards[j].AccountID
+	})
+
+	m["paidStakingRewards"] = paidStakingRewards
+
+	m["prngBytes"] = hex.EncodeToString(record.PrngBytes)
+	m["prngNumber"] = record.PrngNumber
+	m["evmAddress"] = hex.EncodeToString(record.EvmAddress)
+
+	if record.Receipt.TransactionID != nil {
+		receiptJSON, err := record.Receipt.MarshalJSON()
+		if err != nil {
+			return nil, err
+		}
+		var receiptObj map[string]interface{}
+		if err := json.Unmarshal(receiptJSON, &receiptObj); err != nil {
+			return nil, err
+		}
+		m["receipt"] = receiptObj
+	}
+	m["children"] = record.Children
+	m["duplicates"] = record.Duplicates
+
+	receiptBytes, err := record.Receipt.MarshalJSON()
+	if err != nil {
+		return nil, err
+	}
+	var receiptInterface interface{}
+	err = json.Unmarshal(receiptBytes, &receiptInterface)
+	if err != nil {
+		return nil, err
+	}
+	m["receipt"] = receiptInterface
```

Comment:
This should work with just
```suggestion
	m["receipt"] = record.Receipt
```